core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/workspace/WorkspaceStatus.java

```diff
@@ -31,50 +23,44 @@
 public enum WorkspaceStatus {
 
     /**
-     * Workspace considered as starting if and only if its dev-machine is booting(creating).
+     * Workspace considered as starting if and only if its active environment is booting.
      *
      * <p>Workspace becomes starting only if it was {@link #STOPPED}.
      * The status map:
      * <pre>
      *  STOPPED -> <b>STARTING</b> -> RUNNING (normal behaviour)
      *  STOPPED -> <b>STARTING</b> -> STOPPED (failed to start)
      * </pre>
-     *
-     * @see MachineStatus#CREATING
      */
     STARTING,
 
     /**
-     * Workspace considered as running if and only if its dev-machine was successfully started and it is running.
+     * Workspace considered as running if and only if its environment is running.
      *
      * <p>Workspace becomes running after it was {@link #STARTING}.
      * The status map:
      * <pre>
      *  STARTING -> <b>RUNNING</b> -> STOPPING (normal behaviour)
      *  STARTING -> <b>RUNNING</b> -> STOPPED (dev-machine was interrupted)
```

Comment:
As workspace status is not related to `dev-machine` start anymore, you have to fix these _flows_ as well